app/assets/stylesheets/app/consent_form_preview.scss

```diff
@@ -0,0 +1,40 @@
+.consent {
```

Comment:
I know the rule of 3 thing but its good to create generic styles rather than styles tied to a single use case.

Some of these styles could be renamed and don't necessarily have to happen in a consent class.